### DISCUSSION TITLE
Observability for LLM Calls?

### DISCUSSION THREAD

**Discussion by lifan0127** [microsoft/teams-sdk#1573] [Announcements]
Hello, Is there a way to monitor the specific LLM calls made by the library?

**Comment by corinagum** [comment]
Hi @lifan0127, this is not available currently, but is a highly sought-after feature request. Our PMs are discussing it for our roadmap. Please keep an eye out for more information :)

**Reply by corinagum** [reply]
@govil1811  Please feel free to add any information.
